### ISSUE TITLE
Option to ping on a specific text channel a specific role for people to get pinged to notify them about people inside the voice channel

### ISSUE THREAD

**Issue by MarkKomodo** [Erallie/voicely-ping#2]

**Comment by Erallie** [comment]
@MarkKomodo I agree with this suggestion. Sorry for taking so long to see it. It will be in the plans, but it may take a while for me to get to it because I am currently working on other things.

**Comment by Erallie** [comment]
@MarkKomodo since Voicely Ping has taken on a sort of different function than intended for server owners, I've decided to make a new bot that does exactly this. Sorry for taking so long to implement this.

You can find the bot here: https://github.com/Erallie/voicely-role
Or invite it to your server here: https://discord.com/oauth2/authorize?client_id=1527731147428073493
